### DISCUSSION TITLE
Build ash with "linked" feature on Windows fails

### DISCUSSION THREAD

**Discussion by skibon02** [ash-rs/ash#701] [Q&A] [answered]
Inputs are simple: i have a project that uses ash with "linked" feature. It works fine on Linux.
But when i build it on Windows, it fails while linking rust-vulkan crate:
`note: LINK : fatal error LNK1181: cannot open input file 'vulkan-1.lib'`

I surely have VulkanSDK installed and `vulkan-1.lib` in default folder `C:\VulkanSDK\1.3.236.0\Lib`.

Can I point this folder to ash to find the library there? Or i have to copy this lib manually to my project?

**Comment by Ralith** [comment, marked as answer]
Have you rebooted since installing the SDK?

**Reply by skibon02** [reply]
Thank you, it was the actual problem. I forgot how Windows like to reboot after an installation.
